SMARTMANUFACTURING/dataModel.PredictiveMaintenance/MaintenanceService/examples/example-normalized.json

```diff
@@ -0,0 +1,75 @@
+{
+    "id": "urn:ngsi-ld:dataModel.PredictiveMaintenance:MaintenanceService:maintenanceService01",
+    "type": "MaintenanceService",
+    "machineID": {
+        "type": "Property",
+        "value": "S/N123456789"
+    },
+    "customerID": {
+        "type": "Property",
+        "value": "CUST001"
+    },
+    "machineAddressLocation": {
+        "type": "Property",
+        "value": "123 Factory Street, Anytown, USA"
+    },
+    "hasConfiguration": {
```

Comment:
I have to check that we proper the ListProperties properly, Nothing on your side but we could have